.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,13 @@
+- [ ] I have searched the [issues](https://github.com/pingcap-incubator/cherry-bot/issues) of this repository and believe that this is not a duplicate.
+
+### Related link
+
+- Which project: <!-- TiKV/TiDB/... -->
+
+- PR or Issue Link: <!-- pingcap/tidb#18217 -->
```

Comment:
The example comment may be pingcap-incubator/cherry-bot

Reply:
I think we can keep it. Because the bug reporter always comes from other project.